packages/orbit-components/playroom/snippets/BasicComponents.ts

```diff
@@ -54,6 +54,139 @@ const dialog = `
 
 const heading = `<Heading as="h1" type="title1">Heading</Heading>`;
 
+const linkList = `
+  <LinkList
+    direction="row"
+    spacing="400"
+    useMargin
+  >
+    <TextLink type="secondary">
+      Flight 1
+    </TextLink>
+    <TextLink type="secondary">
+      Flight 2
+    </TextLink>
+    <TextLink type="secondary">
+      Flight 3
+    </TextLink>
+  </LinkList>
+`;
+
+const list = `
+  <List>
+    <ListItem>
+      24,000 locations{' '}
+      <TextLink href="#">
+        around
+      </TextLink>
+      {' '}the globe
+    </ListItem>
```

Comment:
We can keep it much simpler by just having some Text, no need for a link or whatever